src/include/btmem.h

```diff
@@ -813,6 +813,23 @@ struct __wt_page {
 #define WT_PREPARE_LOCKED 2
 #define WT_PREPARE_RESOLVED 3
 
+/* Convert a prepare state to its string representation. */
```

Comment:
We don't usually add functions to regular header files. I think this would be better in btree_inline.h, txn_inline.h or perhaps a new xx_inline.h file

Reply:
Thanks - I've added `str_inline.h` for these, it didn't seem quite right in either of those.